src/lib/sendMessage.js

```diff
@@ -0,0 +1,28 @@
+import lineClient from 'src/webhook/lineClient';
+import lineNotify from 'src/lib/lineNotify';
+
+const notify = async (token, message) => {
+  lineNotify(token, { message: message });
+};
+
+// https://developers.line.biz/en/reference/messaging-api/#send-multicast-message
+const multicast = async (userIdList, message) => {
+  lineClient('/message/multicast', {
+    to: userIdList,
+    messages: [{ type: 'text', text: message }],
+  });
+};
+
+// https://developers.line.biz/en/reference/messaging-api/#send-push-message
+const push = async (userId, message) => {
```

Comment:
I wonder if `push` is used anywhere yet?

Reply:
no, currently we just use `multicast` and `notify`.